subgraphs/uniswap-v3/src/common/updateMetrics.ts

```diff
@@ -88,6 +88,9 @@ export function updateUsageMetrics(
   } else if (usageType == UsageType.SWAP) {
     usageMetricsDaily.dailySwapCount += INT_ONE;
     usageMetricsHourly.hourlySwapCount += INT_ONE;
+  } else if (usageType == UsageType.COLLECT) {
+    usageMetricsDaily.dailyCollectCount += INT_ONE;
+    usageMetricsHourly.hourlyCollectCount += INT_ONE;
```

Comment:
Related to my first comment, should we increase the counter of withdrawals too? Essentially a collection is a withdrawal. If someone expects the default DEX schema or is not aware of collections they might expect money out to be counted as a withdrawal?

Reply:
Yeah - makes sense. I just removed the collect from usage metrics.